batcher/aligned-sdk/src/sdk.rs

```diff
@@ -136,37 +136,17 @@ pub async fn submit_multiple_and_wait_verification(
 pub async fn estimate_fee(
     eth_rpc_url: &str,
     estimate: PriceEstimate,
-) -> Result<U256, errors::MaxFeeEstimateError> {
+) -> Result<U256, errors::FeeEstimateError> {
```

Comment:
needs to be more clear. unclear why a function called `estimate_fee` takes `estimate` as a parameter